owncloud/owncloud.py

```diff
@@ -717,6 +717,34 @@ def delete_user(self, user_name):
 
         raise ResponseError(res)
 
+    def get_user_list(self, user_name):
+        """Gets a lit of users via provisioning API.
+        If you get back an error 999, then the provisioning API is not enabled.
+
+        :param user_name:  substring of username to be checked 
+        :returns: a list of users matching the user_name sub string
+        :raises: ResponseError in case an HTTP error status was returned
+
+        """
+
+        res = self.__make_ocs_request(
+            'GET',
+            self.OCS_SERVICE_CLOUD,
+            'users?search=' + user_name
```

Comment:
Are you suggesting I pass in the user_name to the api requrst as:

user_name.encode ('utf_8')

Shouldn't the api handle that?

We should probably make the change made throughout the file for all strings being passed into the api (group names, file names, user names, etc) in which case do we want a new branch for this work?